[15-May-2023 01:27:21 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:350
Stack trace:
#0 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/RegisterErrorViewPaths.php(18): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(659): Illuminate\Foundation\Exceptions\RegisterErrorViewPaths->__invoke()
#2 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(640): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#3 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(571): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#4 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(477): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
#5 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(394): Illuminate\Foundation\Exceptions\Handler->renderExceptionResponse(Object(Illuminate\Http\Request), Object(ValueError))
#6 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(503): Illuminate\Foundation\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(ValueError))
#7 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(148): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(ValueError))
#8 /home/customer/www/demo.purplestairs.com/public_html/index.php(51): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#9 {main}
  thrown in /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 350
[15-May-2023 01:27:32 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:350
Stack trace:
#0 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/RegisterErrorViewPaths.php(18): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(659): Illuminate\Foundation\Exceptions\RegisterErrorViewPaths->__invoke()
#2 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(640): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#3 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(571): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#4 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(477): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
#5 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(394): Illuminate\Foundation\Exceptions\Handler->renderExceptionResponse(Object(Illuminate\Http\Request), Object(ValueError))
#6 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(503): Illuminate\Foundation\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(ValueError))
#7 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(148): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(ValueError))
#8 /home/customer/www/demo.purplestairs.com/public_html/index.php(51): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#9 {main}
  thrown in /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 350
[15-May-2023 01:27:34 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:350
Stack trace:
#0 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/RegisterErrorViewPaths.php(18): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(659): Illuminate\Foundation\Exceptions\RegisterErrorViewPaths->__invoke()
#2 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(640): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#3 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(571): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#4 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(477): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
#5 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(394): Illuminate\Foundation\Exceptions\Handler->renderExceptionResponse(Object(Illuminate\Http\Request), Object(ValueError))
#6 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(503): Illuminate\Foundation\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(ValueError))
#7 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(148): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(ValueError))
#8 /home/customer/www/demo.purplestairs.com/public_html/index.php(51): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#9 {main}
  thrown in /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 350
[15-May-2023 01:27:36 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:350
Stack trace:
#0 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/RegisterErrorViewPaths.php(18): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(659): Illuminate\Foundation\Exceptions\RegisterErrorViewPaths->__invoke()
#2 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(640): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#3 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(571): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#4 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(477): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
#5 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(394): Illuminate\Foundation\Exceptions\Handler->renderExceptionResponse(Object(Illuminate\Http\Request), Object(ValueError))
#6 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(503): Illuminate\Foundation\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(ValueError))
#7 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(148): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(ValueError))
#8 /home/customer/www/demo.purplestairs.com/public_html/index.php(51): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#9 {main}
  thrown in /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 350
[15-May-2023 01:27:38 UTC] PHP Fatal error:  Uncaught RuntimeException: A facade root has not been set. in /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php:350
Stack trace:
#0 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/RegisterErrorViewPaths.php(18): Illuminate\Support\Facades\Facade::__callStatic('replaceNamespac...', Array)
#1 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(659): Illuminate\Foundation\Exceptions\RegisterErrorViewPaths->__invoke()
#2 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(640): Illuminate\Foundation\Exceptions\Handler->registerErrorViewPaths()
#3 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(571): Illuminate\Foundation\Exceptions\Handler->renderHttpException(Object(Symfony\Component\HttpKernel\Exception\HttpException))
#4 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(477): Illuminate\Foundation\Exceptions\Handler->prepareResponse(Object(Illuminate\Http\Request), Object(Symfony\Component\HttpKernel\Exception\HttpException))
#5 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Exceptions/Handler.php(394): Illuminate\Foundation\Exceptions\Handler->renderExceptionResponse(Object(Illuminate\Http\Request), Object(ValueError))
#6 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(503): Illuminate\Foundation\Exceptions\Handler->render(Object(Illuminate\Http\Request), Object(ValueError))
#7 /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(148): Illuminate\Foundation\Http\Kernel->renderException(Object(Illuminate\Http\Request), Object(ValueError))
#8 /home/customer/www/demo.purplestairs.com/public_html/index.php(51): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#9 {main}
  thrown in /home/customer/www/demo.purplestairs.com/app/vendor/laravel/framework/src/Illuminate/Support/Facades/Facade.php on line 350
[17-May-2023 14:32:40 Asia/Colombo] PHP Fatal error:  Constant expression contains invalid operations in /home/customer/www/demo.purplestairs.com/app/app/Http/Livewire/CompanyDashboard.php on line 56
